android-components/components/feature/webcompat/src/main/assets/extensions/webcompat/manifest.json

```diff
@@ -2,8 +2,13 @@
   "manifest_version": 2,
   "name": "Mozilla Android Components - Web Compatibility Interventions",
   "description": "Urgent post-release fixes for web compatibility.",
+<<<<<<< HEAD
   "version": "106.0.0",
```

Comment:
I think here we just want to make increase minor version to `"106.2.0"` right?

Reply:
Yes, just any version-number bump will do, so just incrementing the middle number is fine.

Reply:
I just pushed to update the last digit to be more consistent as it's bug fix 😅  `106.0.1` .